### PR TITLE
Atlas, Clarion, Horizon, Manta, and Oshan Spacemas maps

### PR DESCRIPTION
**Pull request by nefarious6th**
## About the PR 
Adds Spacemas map variants for Atlas, Clarion, Horizon, Manta, and Oshan maps.

## Why's this needed? 
Spacemas map variants to combat a terrible lack of Spacemas cheer!

## Changelog

```
(u)Nefarious6th:
(+) Adds Spacemas variants of Atlas, Clarion, Horizon, Manta, and Oshan maps
```

### CODE REVIEW

**Comment by RSG250** [comment]
hope you didnt forget the stocking

**Comment by nefarious6th** [comment]
To clarify, stockings are in! I may or may not have forgotten to bring the eggnog though...

**Review by flrsh — APPROVED** [review]

**Comment by nefarious6th** [comment]
Also, I forgot to add it prior, but I know Kubius's ephermeral ornaments are in; these map files do have some swapped out tiles on each of them that wouldn't be toggled by the ephermeral marker.

**Review by ZeWaka — CHANGES_REQUESTED** [review]
Just adding the maps won't make them used. Please either add the ephermal thingies to the original maps or edit the code to use these instead.

File: maps/config/horizon.dm [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // #ifdef XMAS
-// #include "..\warwip\atlas_horizon_xmas_2019.dmm"
+// #include "..\horizon_xmas.dmm"
```
Comment (ZeWaka) [inline maps/config/horizon.dm:2]: you'll need to un comment the includes

**Review by ZeWaka — CHANGES_REQUESTED** [review]

File: maps/config/horizon.dm (hunk 1 above)
Comment (nefarious6th) [reply maps/config/horizon.dm:2]: Got it! Thank you for your patience on this PR!

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**